backend/api/Services/AreaService.cs

```diff
@@ -0,0 +1,247 @@
+﻿using Api.Controllers.Models;
+using Api.Database.Context;
+using Api.Database.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace Api.Services
+{
+    public interface IAreaService
+    {
```

Comment:
There are several public functions in this class that have not been included in the interface. Consider if they should be included or made private.